shotgun_api3/shotgun.py

```diff
@@ -593,6 +599,17 @@ def find(self, entity_type, filters, fields=None, order=None,
         :param include_archived_projects: Optional, flag to include entities
         whose projects have been archived
 
+        :param additional_filter_presets: Optional list of presets to
+        further filter the result set, list has the form:
+        [ { preset_name: <preset_name>, <optional_preset parameters> } ],
+
+        Note that these filters are ANDed together and ANDed with the 'filter'
+        argument.
+
+        For details on supported presets and the format of this parameter,
+        please consult the API documentation:
+        https://github.com/shotgunsoftware/python-api/wiki
```

Comment:
actually, I'm adding a new section for it so (I think) the link is going to be:
https://github.com/shotgunsoftware/python-api/wiki/Reference%3A-Additional_Filter_Presets-Syntax
Made the change. Will keep an eye on it and will test/update once the wiki changes are committed.